packages/office-ui-fabric-react/src/components/Nav/Nav.types.ts

```diff
@@ -203,8 +204,7 @@ export interface INavLink {
   target?: string;
 
   /**
-   * Point to the parent node key.  This is used in EditNav when move node from sublink to
-   *   parent link vs vers.
+   * @deprecated
```

Comment:
Same thing here

Reply:
Incidentally why is this deprecated?

Reply:
This isn't referenced anywhere in the Nav class itself, and the comment indicated it was for internal use (and incorrectly included in the public interface a long time ago).

Reply:
This is terrible :(  Thanks for catching.

Reply:
It would be great if you add a comment after deprecated to say this prop si not used anywhere and needs to be cleaned up.

Reply:
Fixed